apps/web/src/data/en/index.ts

```diff
@@ -1043,6 +1043,11 @@ export const loggedInData = {
           alternativeAnswers: 'Alternative answers',
           acceptMathEquivalents: 'Accept all equivalent mathematical values',
         },
+        blanksExerciseDragAndDrop: {
+          title: 'Drag&Drop exercise',
+          description:
+            'Create a fill-in-the-gap text or a table where the correct answers are inserted via drag and drop.',
```

Comment:
```suggestion
            'Create a fill in the blanks exercise (text or table) where learners add the answers via drag and drop.',
```

btw: the "fill-in-the-gap" part is also incorrect in the original string above.

Reply:
The strings come from [this document](https://docs.google.com/document/d/1Jqd3qqRtBX-J3dKwC7-rohDqZGTSGyNNvPq5XXaKllY/edit#heading=h.ubjbe6nuxwf2) and were discussed with @GregorZupan , so I'll leave this to him.

Reply:
Proposal accepted! 

I'll edit it in my tooltip document

Reply:
I also changed the german copy a bit